centos_ci.sh

```diff
@@ -224,6 +278,16 @@ else
     cd gopath/src/github.com/minishift/minishift
     make gen_adoc_tar IMAGE_UID=$(id -u)
     docs_tar_upload $PASS
+  elif [[ "$JOB_NAME" = "minishift-release" ]]; then
+    GIT_USER="minishift-bot"
+    prepare;
+    cd gopath/src/github.com/minishift/minishift
+    bump_to_release_version $GIT_USER;
+    make release
+    add_release_notes $GIT_USER;
+    # Generate docs tar and upload
```

Comment:
@hferentschik I believe you were talking about these two steps here in your last comment.